packages/fxa-shared/oauth/constants.js

```diff
@@ -5,7 +5,9 @@
 module.exports = {
   OAUTH_SCOPE_OLD_SYNC: 'https://identity.mozilla.com/apps/oldsync',
   OAUTH_SCOPE_SESSION_TOKEN: 'https://identity.mozilla.com/tokens/session',
-  OAUTH_SCOPE_NEWSLETTERS: 'https://identity.mozilla.com/accounts/newsletters',
+  OAUTH_SCOPE_NEWSLETTERS: 'https://identity.mozilla.com/account/newsletters',
+  OAUTH_SCOPE_SUBSCRIPTIONS:
+    'https://identity.mozilla.com/account/subscriptions',
```

Comment:
Oh I also moved this scope here so that identical string isn't repeated in multiple places.